doc/sphinx/proof-engine/ltac.rst

```diff
@@ -2504,7 +2501,7 @@ performance issue.
    Set Ltac Profiling.
    tac.
    Show Ltac Profile.
-   Show Ltac Profile "lia".
+   Show Ltac Profile "mytauto".
```

Comment:
This will not behave as expected, because `mytauto` does not appear in the profile. (No idea if this the expected behavior.)